src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -140,9 +141,16 @@ uct_ib_mlx5_devx_reg_ksm_data_mt(uct_ib_mlx5_md_t *md, int atomic,
     void *mr_address       = address;
     size_t list_size       = ksm_data->mr_num;
     ucs_status_t status;
+    struct ibv_mr **mr;
     char *in;
     void *klm;
-    int i;
+
+    if (reason != NULL) {
```

Comment:
I think it worth printing this information even if there is no reason defined. WDYT?